components/integration/base/external-secrets/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- snyk-shared-secret.yaml
+namespace: tekton-ci 
```

Comment:
the `tekton-ci` namespace exists only on the Redhat cluster. You would have to create an overlay for that cluster and adjust the integration application set accordingly (you can see the gitops components as an example).

Reply:
Thanks for your comments! I'll add the external secret to tekton-ci component directly.